### ISSUE TITLE
Remove unused 'notifycompat' and 'notifyevents' from UserProfile

### ISSUE THREAD

**Issue by EnTeQuAk** [mozilla/addons-server#8172]
Looks like they were added in 2010 in de8c6ef38af16ddbe4e761ab4db1f0c2eeb9b7a6 but never used.

**Comment by championshuttler** [comment]
@EnTeQuAk I would like to work on it

**Comment by championshuttler** [comment]
@eviljeff Sure I will try to do it :)

**Comment by AlexandraMoga** [comment]
@eviljeff is this essentially a duplicate of #8274 then?

**Comment by eviljeff** [comment]
@AlexandraMoga no, this issue removed the columns and references from the code; #8274 will tidy up the database.

**Comment by eviljeff** [comment]
QA is just user profiles still work and you can log in; create a user; etc.  Things you may test anyway.

**Comment by ioanarusiczki** [comment]
Verified fixed on AMO dev with FF60(Win10).
Tested: login, create a user, user profile pages - view profile, edit profile.
